bruno/collections/Rafiki/Rafiki Admin APIs/Get Accounting Ledger Transfers.bru

```diff
@@ -0,0 +1,49 @@
+meta {
+  name: Get Accounting Ledger Transfers
+  type: graphql
+  seq: 1
+}
+
+post {
+  url: {{RafikiGraphqlHost}}/graphql
+  body: graphql
+  auth: none
+}
+
+body:graphql {
+  query AccountingTransfers($id: String!, $limit: Int!) {
+    accountingTransfers(id: $id, limit: $limit) {
+      debits {
+        id
+        debitAccount
+        creditAccount
+        amount
+        transferType
+        ledger
+        createdAt
+      }
+      credits {
+        id
+        debitAccount
+        creditAccount
+        amount
+        transferType
+        ledger
+        createdAt
+      }
+    }
+  }
+  
+}
+
+body:graphql:vars {
+  {
+      "id": "{{outgoingPaymentId}}"
+  }
+}
```

Comment:
getting an error without the limit var

```suggestion
body:graphql:vars {
  {
      "id": "{{outgoingPaymentId}}",
      "limit": 100
  }
}
```
Actually my first though was not to require the limit in the GQL schema and set some default limit on our backend. This is what out paginated resolvers do. However, since we aren't returning any sort of page info back I think that behavior would be unclear so maybe requiring it is the way to go for now - just something to keep in mind if we do implement paging here.

Reply:
@BlairCurrey Apologies, I have made the limit optional, then also mentioning the limit in the docs. 

<img width="1256" alt="image" src="https://github.com/user-attachments/assets/fc8dc261-bf79-4b0f-a363-87f74d9b7e42">